<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrame</class>
 <widget class="QWidget" name="MainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="image1_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    border: 1px solid rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Image1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="realtime_label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>801</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    border: 1px solid rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>real-time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>900</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-image: url(&quot;D:/grade 11/AI&amp;ML club/UI/button.png&quot;);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>330</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Class:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>380</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Confidence:</string>
   </property>
  </widget>
  <widget class="QLabel" name="class_label">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>330</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </widget>
  <widget class="QLabel" name="conf_label">
   <property name="geometry">
    <rect>
     <x>1640</x>
     <y>380</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>96.84%</string>
   </property>
  </widget>
  <widget class="QLabel" name="predicted1_label">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>450</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    border: 1px solid rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Image1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Waiting ...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>280</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>bg_squid.jpg</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>image1_label</zorder>
  <zorder>realtime_label</zorder>
  <zorder>start_button</zorder>
  <zorder>label1</zorder>
  <zorder>label2</zorder>
  <zorder>class_label</zorder>
  <zorder>conf_label</zorder>
  <zorder>predicted1_label</zorder>
  <zorder>status_label</zorder>
  <zorder>time_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
